src/pages/workspace/accounting/qbo/import/QuickbooksChartOfAccountsPage.tsx

```diff
@@ -1,74 +1,63 @@
 import React from 'react';
-import {View} from 'react-native';
 import ConnectionLayout from '@components/ConnectionLayout';
 import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
-import OfflineWithFeedback from '@components/OfflineWithFeedback';
-import Switch from '@components/Switch';
 import Text from '@components/Text';
 import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
 import * as Connections from '@libs/actions/connections';
+import * as ErrorUtils from '@libs/ErrorUtils';
+import {settingsPendingAction} from '@libs/PolicyUtils';
 import type {WithPolicyProps} from '@pages/workspace/withPolicy';
 import withPolicyConnections from '@pages/workspace/withPolicyConnections';
-import variables from '@styles/variables';
+import ToggleSettingOptionRow from '@pages/workspace/workflows/ToggleSettingsOptionRow';
+import {clearQBOErrorField} from '@userActions/Policy/Policy';
 import CONST from '@src/CONST';
 
 function QuickbooksChartOfAccountsPage({policy}: WithPolicyProps) {
     const {translate} = useLocalize();
     const styles = useThemeStyles();
     const policyID = policy?.id ?? '-1';
-    const {enableNewCategories, pendingFields} = policy?.connections?.quickbooksOnline?.config ?? {};
+    const qboConfig = policy?.connections?.quickbooksOnline?.config;
 
     return (
         <ConnectionLayout
+            policyID={policyID}
             displayName={QuickbooksChartOfAccountsPage.displayName}
             headerTitle="workspace.accounting.accounts"
             title="workspace.qbo.accountsDescription"
             accessVariants={[CONST.POLICY.ACCESS_VARIANTS.ADMIN]}
-            policyID={policyID}
             featureName={CONST.POLICY.MORE_FEATURES.ARE_CONNECTIONS_ENABLED}
             contentContainerStyle={[styles.pb2, styles.ph5]}
             connectionName={CONST.POLICY.CONNECTIONS.NAME.QBO}
         >
-            <View style={[styles.flexRow, styles.mb4, styles.alignItemsCenter, styles.justifyContentBetween]}>
-                <View style={styles.flex1}>
-                    <Text fontSize={variables.fontSizeNormal}>{translate('workspace.accounting.import')}</Text>
-                </View>
-                <View style={[styles.flex1, styles.alignItemsEnd, styles.pl3]}>
-                    <Switch
-                        accessibilityLabel={translate('workspace.accounting.accounts')}
-                        isOn
-                        disabled
-                        onToggle={() => {}}
-                    />
-                </View>
-            </View>
+            <ToggleSettingOptionRow
+                title={translate('workspace.accounting.import')}
+                switchAccessibilityLabel={translate('workspace.accounting.accounts')}
```

Comment:
accessibility label is different than the previous one

```
accessibilityLabel={translate('workspace.accounting.accounts')}
```

should it?

Reply:
it was like this before the refactor, was it wrong then? 🤔

Reply:
See [this](https://github.com/Expensify/App/pull/46511/files/0cff3d994e289bc54912dd94d2af620c97f5ac7c#r1706607401) comment, sorry 😅